<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1684313790431' stop='1684313852662'>
    <name>Sign Up to FLAT6LABS</name>
    <title>Sign Up to FLAT6LABS</title>
    <test-cases>
        <test-case start='1684313790431' status='failed' stop='1684313796254'>
            <name>Should visit the login page correctly</name>
            <title>Should visit the login page correctly</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Timed out retrying after 4000ms: expected 'https://ignite.mobelite-digital.info/sign-in' to include 'https://ignite.mobelite-digital.info/ '</message>
                <stack-trace>AssertionError: Timed out retrying after 4000ms: expected 'https://ignite.mobelite-digital.info/sign-in' to include 'https://ignite.mobelite-digital.info/ '
                    at Application.CheckURL (webpack:///./cypress/PageObjects/Application/Application.js:3:17)
                    at Context.eval (webpack:///./cypress/e2e/CheckApplication.cy.js:19:18)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>